org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/IVirtualMachineManagerProvider.java

```diff
@@ -0,0 +1,5 @@
+package org.eclipse.jdt.ls.debug.adapter;
+
+public interface IVirtualMachineManagerProvider {
```

Comment:
current implementation is that debugger used the same virtualmachinemanager as language server, which is `org.eclipse.jdi.Bootstrap.virtualMachineManager`, we can encapsulate this.